### PR TITLE
feat(targeting-ftux): add targeting and ftux context

### PR DESCRIPTION
**Pull request by azuruce**

### CODE REVIEW

**Comment by boxcla** [comment]
Hi @azuruce, thanks for the pull request. Before we can merge it, we need you to sign our Contributor License Agreement. You can do so electronically here: http://opensource.box.com/cla

Once you have signed, just add a comment to this pull request saying, "CLA signed". Thanks!

**Comment by azuruce** [comment]
CLA signed

**Comment by azuruce** [comment]
next commit #1845

File: src/features/targeting/types.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+// @flow
+export type FtuxContextValue = {};
+
+export type TargetingContextValue = {};
```
Comment (priyajeet) [inline src/features/targeting/types.js]: Probably make these more concrete eventually?

File: src/features/targeting/index.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+// @flow
+export * from './contexts';
+
+export * from './types';
```
Comment (priyajeet) [inline src/features/targeting/index.js]: Will this export the type without type keyword? When downcompiled, it won't export anything btw.

File: src/features/targeting/types.js (hunk 1 above)
Comment (azuruce) [reply src/features/targeting/types.js]: yes

File: src/features/targeting/index.js (hunk 2 above)
Comment (azuruce) [reply src/features/targeting/index.js]: it exports types correctly.

**Pull request merged**